trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/compute_wing_induced_velocity.py

```diff
@@ -12,11 +13,14 @@
 import numpy as np 
 
 ## @ingroup Methods-Aerodynamics-Common-Fidelity_Zero-Lift
-def compute_wing_induced_velocity(VD,mach):
+def compute_wing_induced_velocity(VD,mach,computed_in_VLM=False):
     """ This computes the induced velocities at each control point of the vehicle vortex lattice 
 
     Assumptions: 
     Trailing vortex legs infinity are alligned to freestream
+    
+    VORLAX's EW only ever needs to be calculated for VLM to match up with VORLAX's boundary
```

Comment:
Does this mean it's only calculated when matching VORLAX boundary conditions is the goal?

Reply:
Yeah. I'll make that more explicit

Reply:
I generalized the option input "computed_in_VLM=False" to just "compute_EW=False" and explained that EW is typically redundant unless requested by the user since C_mn provides the same information but in body-frame